### PR TITLE
Updated ReadMe and added Udemy and DailyDev

### PR DESCRIPTION
**Pull request by NJWasTaken**
Updated **ReadME** to include 1 line python commands to run sherlock without direct installation.
Added **Udemy** and **DailyDev** to the list of websites.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -96,6 +96,14 @@ optional arguments:
...
 ```
 
+In case installation fails:
+```bash
+python -m sherlock_project user1 user2 user3 
+```
+or
+```bash
+python3 -m sherlock_project user1 user2 user3
+```
```
Comment (ppfeister) [inline docs/README.md:106]: Could you explain this section please?

Python's gotta get it from somewhere, and that would presumably be by installation (which either failed or didn't)

File: sherlock_project/resources/data.json [hunk 2]
```diff
@@ -613,6 +613,13 @@
     "urlMain": "https://www.dailymotion.com/",
     "username_claimed": "blue"
   },
+  "DailyDev":{
+    "errorType": "message",
+    "errorMsg":"Page not found.",
+    "url": "https://app.daily.dev/{}",
+    "urlMain": "https://app.daily.dev/",
+    "username_claimed": "nj05"
+  },
```
Comment (ppfeister) [inline sherlock_project/resources/data.json:622]: Returns positive for all queries
`errorType` of `message` should be a last resort behind status_code due to l10n issues

File: sherlock_project/resources/data.json [hunk 3]
```diff
@@ -2777,6 +2784,13 @@
     "urlMain": "https://www.toster.ru/",
     "username_claimed": "adam"
   },
+  "udemy": {
+    "errorMsg": "Page not found",
+    "errorType": "message",
+    "url":"https://www.udemy.com/user/{}",
+    "urlMain": "https://www.udemy.com/",
+    "username_claimed": "adfadf"
+  },
```
Comment (ppfeister) [inline sherlock_project/resources/data.json:2793]: Returns positive for all queries
`errorType` of `message` should be a last resort behind status_code due to l10n issues

**Review by ppfeister — CHANGES_REQUESTED** [review]

File: sherlock_project/resources/data.json [hunk 4]
```diff
@@ -2777,6 +2784,13 @@
     "urlMain": "https://www.toster.ru/",
     "username_claimed": "adam"
   },
+  "udemy": {
```
Comment (ppfeister) [inline sherlock_project/resources/data.json:2787]: Also please capitalize for UX (minor bikeshed maybe)

File: docs/README.md (hunk 1 above)
Comment (NJWasTaken) [reply docs/README.md:106]: I'm not 100% sure why only this method works for me, but I'm guessing even if the installation is somewhat successful, running it by directly calling the module with Python bypasses permission or setup issues.

**Pull request closed without merging**